reproman/ui/progressbars.py

```diff
@@ -89,7 +89,8 @@ def finish(self):
                 self._pbar.finish()
             super(progressbarProgressBar, self).finish()
 
-    progressbars['progressbar'] = progressbarProgressBar
+	# TODO(asmacdo) progress bar fails for some reason, skipping for now
+    # progressbars['progressbar'] = progressbarProgressBar
```

Comment:
I think we removed support for the `progressbar` -- only `tqdm` . But I am not sure why we would fail here at all since we just should skip if no `progressbar` module... anyways -- I would recommend to just strip off `progressbar` support and rely only on `tqdm` and make it the one `setup.py` installs

Reply:
removed progressbar class!